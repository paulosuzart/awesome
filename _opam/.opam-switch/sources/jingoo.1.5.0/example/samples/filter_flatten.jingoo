{%- set test = [[1,2],[3],4] | flatten -%}
test[0]: {{ test[0] }}
test[1]: {{ test[1] }}
test[2]: {{ test[2] }}
test[3]: {{ test[3] }}
