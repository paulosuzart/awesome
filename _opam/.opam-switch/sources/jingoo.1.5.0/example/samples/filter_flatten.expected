test[0]: 1
test[1]: 2
test[2]: 3
test[3]: 4
